src/components/RJST/components/Table/TableActions.tsx

```diff
@@ -28,6 +39,9 @@ export const TableActions = <T extends object>({
 	onColumnPreferencesChange,
 }: TableActionsProps<T>) => {
 	const [anchorEl, setAnchorEl] = React.useState<HTMLElement>();
+	const theme = useTheme();
+	const { state: analyticsState } = useAnalyticsContext();
+	const matches = useMediaQuery(theme.breakpoints.up('sm'));
```

Comment:
nit:
```suggestion
	const mobile = useMediaQuery(theme.breakpoints.down('md'));
```

Like we do everywhere else?

Reply:
I can agree with doing always the same thing everywhere, but atm we use more theme.breakpoints.up('sm') then down('md') in ui-shared-compoennts. Let me open a different PR to add more consistency on that